service/src/main/java/com/commercetools/pspadapter/payone/domain/payone/model/klarna/KlarnaAuthorizationRequest.java

```diff
@@ -0,0 +1,20 @@
+package com.commercetools.pspadapter.payone.domain.payone.model.klarna;
+
+import com.commercetools.pspadapter.payone.config.PayoneConfig;
+import com.commercetools.pspadapter.payone.domain.ctp.PaymentWithCartLike;
+import com.commercetools.pspadapter.payone.domain.payone.model.common.RequestType;
+
+/**
+ * <b>Caveat:</b> according to Payone discussion this request should not be used by the shops, because in this case
+ * <i>capture</i> state is activated automatically.
+ * Preferable way is to make <i>{@link KlarnaPreauthorizationRequest preauthorization}</i> request and then
+ * make direct <i>capture</i>, when the goods are packaged, shipped or delivered.
+ */
+public class KlarnaAuthorizationRequest extends BaseKlarnaRequest {
```

Comment:
it's not recommended for on-line shops, but still can be use by other suppliers/consumers (warehouses, robots, export/import jobs and so on)